src/main/resources/interface-specification.yml

```
@@ -4300,13 +4300,13 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  /tenants/{tenantId}/attributes/{attributeId}/updateRenewalStrategy:
+  /tenants/{tenantId}/attributes/{attributeId}: #We have to inform the frontend about this change
```

If this detail is not already in the task, can you please add a comment there?
```suggestion
  /tenants/{tenantId}/attributes/{attributeId}:
```